src/Altinn.Common/Altinn.Common.AccessToken/Altinn.Common.AccessTokenClient/Altinn.Common.AccessTokenClient.csproj

```diff
@@ -31,6 +31,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.KeyVault" Version="3.0.5" />
     <PackageReference Include="Microsoft.Azure.Services.AppAuthentication" Version="1.6.2" />
+    <PackageReference Include="Microsoft.Extensions.Caching.Abstractions" Version="6.0.0" />
     <!-- Upgrade to 5.x.x causes function-apps to fail. -->
```

Comment:
if you could delete line 34 as well that would be great. comment no longer applies